src/main/java/frc/robot/OI.java

```diff
@@ -15,7 +15,7 @@ public class OI {
 
         private static final int kOrientationButtonID = 1; //1 Button, Toggle swerve orientation
         private static final int kZeroButtonID = 2; //2 Button, Zero the gyroscope
-
+        private static final int kParkButtonID = 5; //5 Button, park the robot
```

Comment:
What button is the 5 button on the controller? I believe there are actual 1 and 2 buttons on our controller, which is why I let Zane get away with the comments above. But I don't think there is a 5 button...

Reply:
Zane said I could just leave it so I assume he might be working on it

Reply:
Josh added comments about all of the button locations